tests/snuba/eventstream/test_eventstream.py

```diff
@@ -20,6 +20,37 @@ def setUp(self):
         self.kafka_eventstream = KafkaEventStream()
         self.kafka_eventstream.producer = Mock()
 
+    def __build_event(self, timestamp):
```

Comment:
What determines whether or not a method uses one or two `_`?

Reply:
`__` gets name mangling applied, `_` doesn't. We have no consistent policy on that but `__` definitely means: do not override in subclasses.

Reply:
Totally forgot that python did that.